docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
```

Comment:
I'm also not posisitve about using docker-compose for a single contaainer setup...    Not every environment haas docker-compose I seem to recall...   Isn't docker-compose like a `pip install docker-compose` thing?

Reply:
@epugh it's optional. you don't have to use it if you don't want to. but it's there if you do.